web/chat/chat-thread-list.react.js

```diff
@@ -26,24 +28,34 @@ type Props = {|
 |};
 class ChatThreadList extends React.PureComponent<Props> {
   static propTypes = {
+    filterThreads: PropTypes.func.isRequired,
+    emptyItem: PropTypes.element,
```

Comment:
This should be a `PropTypes.elementType` (corresponds to `React.ComponentType`), not a `PropTypes.element` (corresponds to `React.Element`)